en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-260-to-300.md

```diff
@@ -37,314 +320,1598 @@ Follow the instructions below to move all the existing API Manager configuration
+    sh worker.sh
 
-    You need to use the `statTable` migration property for table wise migration.
+    cd <API-M_ANALYTICS_3.0.0_HOME>/bin
+    sh dashboard.sh
+    ```
 
-    For example, if you need to migrate only the `ApiPerDestinationAgg` and `ApiResPathPerApp` tables use the following command:
+!!! note
+    If you have developed any custom dashboards in API-M 2.6.0 Analytics using Stream Processor, you will be able to use the same in API-M Anaytics 3.0.0 as well. If you require any guidance regarding this, you can contact [WSO2 Support](https://support.wso2.com/jira/secure/Dashboard.jspa).
 
-        !!! note
-    When you need to migrate multiple tables, the comma separated table names should without spaces before or after the comma as shown below.
+##### Step 1.2.2 - Configure WSO2 API-M 3.0.0 for Analytics
 
+Follow the instructions below to configure WSO2 API Manager for the WSO2 API-M Analytics migration in order to migrate the statistics related data.
 
-    ``` java
-        sh wso2server.sh -DmigrateStats=true -DstatTable=ApiPerDestinationAgg,ApiResPathPerApp
-    ```
+1.  Configure the following datasources in the `<API-M_3.0.0_HOME>/repository/conf/deployment.toml` file.
 
-        !!! info
-    If the default 3 datasource names/name, which you specified above, changed from the details that you specified in the `<API-M_2.6.0_HOME>/repository/conf/datasources/master-datasources.xml` file. Use the following command to pass the values to the migration client.
+    The following is an example of how the configurations should be defined when using MySQL.
 
-    ![](images/icons/grey_arrow_down.png){.expand-control-image} Click here to see a code snippet of the definition of the datasource name...
+    This datasource points to the **previous API-M version's WSO2AM_DB datasource.**
 
     ``` java
-        <jndiConfig>
-            <name>jdbc/WSO2AM_DB</name>
-        </jndiConfig>
+    [database.apim_db]
+    type = "mysql"
+    url = "jdbc:mysql://localhost:3306/am_db"
+    username = "username"
+    password = "password"
     ```
 
-    -   [**Format**](#code)
-    -   [**Example**](#example)
+2.  Enable analytics in WSO2 API-M by setting the following configuration to true in the `<API-M_3.0.0_HOME>/repository/conf/deployment.toml` file.
 
     ``` java
-            sh wso2server.sh -DdataSource=<AM_DATASOURCE_NAME>,<OLD_STAT_DATASOURCE_NAME>,<NEW_STAT_DATASOURCE_NAME>
-    ```
+    [apim.analytics]
+    enable = true
 
-    ``` java
-            sh wso2server.sh -DdataSource=jdbc/WSO2AM_DB,jdbc/WSO2AM_STATS_DB,jdbc/APIM_ANALYTICS_DB
     ```
 
-        !!! warning
-    The following warning message could occur during migration if the relevant consumer key does not exist in the `AM_APPLICATION_KEY_MAPPING` table due to Application deletion.
+3. Restart the WSO2 API-M Server.
 
     ``` java
-        ConsumerKey <consumerKey> does not contain in the AM_APPLICATION_KEY_MAPPING table.
+    cd <API-M_3.0.0_HOME>/bin
```

Comment:
Shall we have tabs with commands for Linux / Windows?

Reply:
+1